lib/SIL/SILVerifier.cpp

```diff
@@ -4302,6 +4302,32 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     }
   }
 
+  void verifyLoadInitialization(SILFunction *F) {
```

Comment:
Please add a comment on the function which says what it does, e.g. catch trivial cases of not initialized alloc_stack locations.